runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -98,6 +105,12 @@ local LanguageTree = {}
 
 LanguageTree.__index = LanguageTree
 
+--- @return integer
+function LanguageTree:_changedtick()
+  --- @type integer
+  return vim.b[self._source].changedtick
```

Comment:
maybe this is generally useful?
```suggestion
function LanguageTree:_buf()
  return vim.b[self._source]
```